### PR TITLE
Single Run Screenshot Test

### PR DESCRIPTION
**Pull request by RaggedyCoder**

### CODE REVIEW

**Comment by kajornsakp** [comment]
@RaggedyCoder seems like I can't approve the PR after left agoda-com org 😂. Let's try with a specific tag on this branch and test it locally.

**Review by techit-ung — APPROVED** [review]

**Comment by RaggedyCoder** [comment]
> @RaggedyCoder seems like I can't approve the PR after left agoda-com org 😂. Let's try with a specific tag on this branch and test it locally.

Can you help me by providing some info. 😅

**Comment by kajornsakp** [comment]
@RaggedyCoder you can add git tag to your forked branch and trigger the build in jitpack.io. Then replace version in snapshot plugin to use the temporary one for testing (and run it multiple times on CI too).

https://jitpack.io/#RaggedyCoder/screenshot-tests-for-android/single-run-task-SNAPSHOT

Something like this
`implementation 'com.github.RaggedyCoder:screenshot-tests-for-android:single-run-task-SNAPSHOT'`

**Pull request merged**
